app/src/main/java/kantwonskids/donationtrackerg14b/controller/LoginActivity.java

```diff
@@ -143,6 +143,7 @@ private void login(User potentialUser) {
 //        Intent intent = new Intent(this, MainActivity.class);
         Intent intent = new Intent(this, MainActivity.class);
         intent.putExtra("CURRENT_USER", (Parcelable)potentialUser);
```

Comment:
ok, so here's where the current user USED to be set. In the old MainActivity, we had the current user come through with the intent, and then we set the user in the MainActivity class.

Since the MainActivity doesn't do anything to the user, there's not really any reason to do this, so I think the way you fixed it works. We might want to delete the call to putExtra() though since we don't use it anymore.